dev/dll/XamlControlsResources.cpp

```diff
@@ -27,7 +27,7 @@ XamlControlsResources::XamlControlsResources()
 
 bool XamlControlsResources::UseLatestStyle()
 {
-    return Version() != winrt::StylesVersion::WinUI_2dot5;
+    return Version() == winrt::StylesVersion::WinUI_2dot5;
```

Comment:
We have a PR to update the test app to use latest style. We should undo this change.

Reply:
Fixed now.